src/main/java/org/broadinstitute/hellbender/tools/spark/sv/ReadClassifier.java

```diff
@@ -74,8 +76,9 @@ private static boolean hasFinalSoftClip( final List<CigarElement> cigarElements,
     }
 
     private static boolean isHighQualityRegion( final byte[] quals, int idx ) {
+        int lowQuals = 0;
         for ( final int end = idx+MIN_SOFT_CLIP_LEN; idx != end; ++idx ) {
-            if ( quals[idx] < MIN_QUALITY ) return false;
+            if ( quals[idx] < MIN_QUALITY && ++lowQuals > MAX_LOW_QUALITY_SCORES ) return false;
```

Comment:
Stylistic point, but I really don't like having a ++ as a prefix operator nested within a clause of a condition.. would you mind blowing this out? eg

'''
if ( quals[idx] < MIN_QUALITY ) {
   lowQuals = lowQuals + 1;
   if ( lowQuals > MAX_LOW_QUALITY_SCORES ) {
           return false;
   }
}
'''

I know it's more lines of code but I find it much easier to read..